book/installation/installation.md

```diff
@@ -21,8 +21,8 @@ The most important requirement is by far the disk, whereas CPU and RAM requireme
 
 |           | Archive Node                          | Full Node                             |
 |-----------|---------------------------------------|---------------------------------------|
-| Disk      | At least 2.2TB (TLC NVMe recommended) | At least 1.2TB (TLC NVMe recommended) |
-| Memory    | 8GB+                                  | 8GB+                                  |
+| Disk      | At least 2.8TB (TLC NVMe recommended) | At least 1.8TB (TLC NVMe recommended) |
+| Memory    | 16GB+                                 | 8GB+                                  |
```

Comment:
I think these recommendations are fine and in line with CL recommendations, e.g. lh recommends 32gb rn.
so this better.